tests/framework/installer/ceph_manifests.go

```diff
@@ -1714,52 +1714,46 @@ spec:
 // GetRookToolBox returns rook-toolbox manifest
 func (m *CephManifestsMaster) GetRookToolBox(namespace string) string {
 	return `apiVersion: v1
-kind: Pod
-metadata:
-  name: rook-ceph-tools
-  namespace: ` + namespace + `
-spec:
-  dnsPolicy: ClusterFirstWithHostNet
-  containers:
-  - name: rook-ceph-tools
-    image: rook/ceph:` + m.imageTag + `
-    imagePullPolicy: IfNotPresent
-    command: ["/tini"]
-    args: ["-g", "--", "/usr/local/bin/toolbox.sh"]
-    env:
-      - name: ROOK_ADMIN_SECRET
-        valueFrom:
-          secretKeyRef:
-            name: rook-ceph-mon
-            key: admin-secret
-    securityContext:
-      privileged: true
-    volumeMounts:
-      - mountPath: /dev
-        name: dev
-      - mountPath: /sys/bus
-        name: sysbus
-      - mountPath: /lib/modules
-        name: libmodules
-      - name: mon-endpoint-volume
-        mountPath: /etc/rook
-  hostNetwork: false
-  volumes:
-    - name: dev
-      hostPath:
-        path: /dev
-    - name: sysbus
-      hostPath:
-        path: /sys/bus
-    - name: libmodules
-      hostPath:
-        path: /lib/modules
-    - name: mon-endpoint-volume
-      configMap:
-        name: rook-ceph-mon-endpoints
-        items:
-        - key: data
-          path: mon-endpoints`
+  kind: Pod
```

Comment:
The whitespace changed on this file too. Can we only remove the obsolete lines instead of updated all of these?

Reply:
I have fixed it please check